net/radius-mac/files/radius-mac.init

```diff
@@ -0,0 +1,208 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2025 Brian 'redbeard' Harrington <redbeard@dead-city.org>
+# This is free software, licensed under the MIT License
+
+START=99
+USE_PROCD=1
+
+PROG=/usr/bin/radius-mac
+CONFIG_BASE_DIR="/etc" # Directory for generated .ini files
+UCI_CONFIG_NAME="radius-mac"
```

Comment:
Do we need to have this variable to use this only twice? In start and  service_trigers? 🤔